ui/src/config.ts

```diff
@@ -1,5 +1,11 @@
+import * as ipc from "../src/ipc";
+
 export const HIDDEN_BRANCHES = ["rad/contributor", "rad/project"];
 export const UPSTREAM_DEFAULT_BRANCH = "main";
 export const GIT_DEFAULT_BRANCH = "master";
 export const NOTIFICATION_TIMEOUT = 8000; // ms
 export const FADE_DURATION = 200;
+
+export function usersGitDefaultBranch(): Promise<string | undefined> {
+  return ipc.usersGitDefaultBranch();
+}
```

Comment:
:information_source: I added this indirection to mean that this is seen as a 'config' property, which happens to be obtained via `ipc`.

Reply:
> I added this indirection to mean that this is seen as a 'config' property, which happens to be obtained via `ipc`.

This is something we might want to reconsider. In addition to an unnecessary indirection this also has the potential to create problems down the line. For example, if the `ipc` module has to be configured, we would like to have that value in `config`, but this would create a circular dependency.

I’d also like to point out that, despite its name, this module has not been there for configuration but just served as a place to collect random constants. In this light I’d like to avoid extending this module further and complicating the module dependency graph.